src/PrestaShopBundle/Controller/Admin/Sell/Catalog/ManufacturerController.php

```diff
@@ -100,6 +100,7 @@ public function indexAction(
             'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
             'manufacturerGrid' => $this->presentGrid($manufacturerGrid),
             'manufacturerAddressGrid' => $this->presentGrid($manufacturerAddressGrid),
+            'alertInfoMessage' => $this->getAlertInfoMessage(),
```

Comment:
I'm wondering if you couldn't use the flash messages for this, instead of adding a special variable and change the template.

Reply:
I thought about it... here are the rasons why I did it this way, don't hesitate to tell me if you disagree:

1/ This is not really a "flash" message, because it is to be displayed permanently, so it shouldn't depend on session's flashbag. It is not supposed to be closed/remove either.

2/ There is currently no type of flash message for this "info" type of message, I used the already existing:
```twig
{{ ps.infotip(alertInfoMessage) }}
```
3/ I think the problem here is the name I gave to this variable and its method, "alertInfoMessage" is misleading, I should give it a more sensical name, like "permanentInfoMessage" and getPermanentInfoMessage.

What do you think?

Reply:
Alright that makes sense! Although I find it weird that you can never hide this message. We introduced in recent versions some help cards (or showcase) in some pages (which can be hidden thanks to cookies), maybe the same feature could have been used (although it's not up to you to decide)

Reply:
About the name indeed it might be more understandable if you rename it, maybe `settingsInfoMessage` or `settingsTipMessage`

Reply:
done ! I gave it this name